instrumentation/liberty/liberty-dispatcher/javaagent/liberty-dispatcher-javaagent.gradle

```diff
@@ -0,0 +1,5 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+dependencies {
```

Comment:
can you add comment in here about why this is separate module from liberty module? (or can they be combined?)

Reply:
I initially had some issues keeping them in the same module, but now that I tried again it worked. I still would prefer to keep them separated as they really don't interact with each other and are in separate class loaders during runtime. Maybe I initially had something using servlet api inside dispatcher code and that caused it to fail to load because servlet api isn't available for dispatcher during runtime, idk.

Reply:
I think this is a very good reason if they are in different class loaders, can you add comment in the gradle file?

Reply:
comment added